lib/src/Base/Common/ResourceMap.cxx

```diff
@@ -666,6 +666,14 @@ void ResourceMap::loadDefaultConfiguration()
   setAsScalar( "OptimizationAlgorithm-DefaultMaximumResidualError", 1.0e-5 );
   setAsUnsignedInteger( "OptimizationAlgorithm-DefaultMaximumEvaluationNumber", 100);
   setAsUnsignedInteger( "OptimizationAlgorithm-DefaultMaximumIterationNumber", 100 );
+  
+  // Dlib optimization parameters //
+  setAsUnsignedInteger( "Dlib-DefaultMaxSize", 10);
+  setAsScalar( "Dlib-DefaultWolfeRho", 0.01);
+  setAsScalar( "Dlib-DefaultWolfeSigma", 0.9);
+  setAsUnsignedInteger( "Dlib-DefaultMaxLineSearchIterations", 100);
+  setAsScalar("Dlib-DefaultInitialTrustRegionRadius", 1.0);
+  
```

Comment:
Same remark here: sort the keys by type, in alphabetical order within each type, Bool before Scalar before String before UnsignedInteger